app/models/invoice_item.rb

```diff
@@ -7,4 +7,9 @@ class InvoiceItem < ApplicationRecord
 
 
   enum status: { "pending" => 0, :packaged => 1, "shipped" =>2 }
+
+  def self.potential_revenue
+    InvoiceItem.joins(invoice: :transactions).where(transactions: {result: 0})
+    .sum("invoice_items.quantity * invoice_items.unit_price")
```

Comment:
I was wondering there what `potential revenue` that kept cropping up was coming from. That's exactly what I was thinking we would need to do during the lesson w/Jamison this morning!